lib/mixins/property-effects.html

```diff
@@ -1442,6 +1442,32 @@
       }
 
       /**
+       * TODO(wahson):
+       * 1、if property is not an object
+       * 2、support for array
+       * @param {object or Array} property
+       * @param {string} path
+       */
+      _initProxy(property, path) {
+        let handler = (path) => {
+          return {
+            set: (obj, prop, value) => {
+              let _path = `${path}.${prop}`;
+              if (value instanceof Object) {
+                value = new Proxy(value, handler(_path));
```

Comment:
Isn't `Proxy` [slow](http://thecodebarbarian.com/thoughts-on-es6-proxies-performance.html)? What about IE support? I wonder if Polymer should use such stuff.